server/src/main/java/io/crate/analyze/AnalyzedStatement.java

```diff
@@ -60,4 +61,15 @@ public interface AnalyzedStatement {
     default List<Symbol> outputs() {
         return null;
     }
+
+    /**
+     * Names for the {@link #outputs()}.
+     * Can match the string representation of {@link Symbol#toColumn()}, or
+     * for statements like `SELECT foo()` it can be the function name.
+     */
```

Comment:
Is this comment really valid? At least it confuses me as I only see `Symbol.toColumn()` calls here.
```suggestion

```

Reply:
The comment was intended from the POV for a consumer of the interface - it is not a description of this specific default implementation.

The new variant is imho not an improvement.

Reply:
Hm I see, looks like the default impl confused me and I did not consider to mucht that this is an interface. After reading it again, it sounds valid and I agree that the new variant won't be better.